src/juxt/grab/alpha/schema.clj

```diff
@@ -653,9 +653,19 @@
         {:errors (::errors result)})))
     result))
 
+
+
+(defmacro flet [bindings & body]
```

Comment:
The only concern I have with this is it's not idiomatic Clojure, so is harder for Clojure developers to read. Let me unpack it and figure out what it's doing and then offer an alternative.